cmd/gazelle/integration_test.go

```diff
@@ -4009,3 +4008,73 @@ package foo
 		},
 	})
 }
+
+func TestPlatformSpecificEmbedsrcs(t *testing.T) {
```

Comment:
Thanks for reviewing.

I agree the testing a little light here, but since the directive parsing code is copied out of `go/build`, I don't feel it's too risky. Once Go 1.16 is the minimum supported version, we may be able to use that code directly.

That said, I do think it's important to cover a bit more, so I added a test for multiple patterns with quotes. Bazel seems to be unhappy with file names containing spaces in `data` (which `//language/go:go_test` relies on), so I just made it an integration test.

I generally have come to prefer integration tests for Gazelle: they tend to be easier to write, are less sensitive to change, provide better coverage, and aren't too expensive. If I had more time, I'd rewrite a lot of the unit tests we already have.